readme.md

```diff
@@ -113,19 +113,236 @@ can be found in our Kubernetes configuration https://github.com/zalando-incubato
 Skipper's Godoc page, https://godoc.org/github.com/zalando/skipper, includes detailed information on these
 topics:
 
-- The Routing Mechanism
+- The [Routing](https://godoc.org/github.com/zalando/skipper/routing) Mechanism
 - Matching Requests
-- Filters - Augmenting Requests
+- [Filters](https://godoc.org/github.com/zalando/skipper/filters) - Augmenting Requests
 - Service Backends
 - Route Definitions
-- Data Sources: Kubernetes
-- Circuit Breakers
+- Data Sources: [ETCD](https://godoc.org/github.com/zalando/skipper/etcd), [Inkeeper API](https://godoc.org/github.com/zalando/skipper/innkeeper), [Kubernetes](https://godoc.org/github.com/zalando/skipper/dataclients/kubernetes), [Route string](https://godoc.org/github.com/zalando/skipper/dataclients/routestring)
+- [Circuit Breakers](https://godoc.org/github.com/zalando/skipper/filters/circuit)
 - Extending It with Customized Predicates, Filters, and Builds
-- Proxy Packages
-- Logging and Metrics
+- [Predicates](https://godoc.org/github.com/zalando/skipper/predicates) - additional predicates to match a route
+- [Proxy Packages](https://godoc.org/github.com/zalando/skipper/proxy)
+- [Logging](https://godoc.org/github.com/zalando/skipper/logging) and [Metrics](https://godoc.org/github.com/zalando/skipper/metrics)
 - Performance Considerations
-- Rate Limiters
-
+- [Rate Limiters](https://godoc.org/github.com/zalando/skipper/filters/ratelimit)
+
+#### 1 Minute Skipper introduction
+
+The following example shows a skipper routes file in eskip format, that has 3 named routes: baidu, google and yandex.
+
+    % cat doc-1min-intro.eskip
+    baidu:
+            Path("/baidu")
+            -> setRequestHeader("Host", "www.baidu.com")
```

Comment:
it's probably fine for this example, but worth to know that with the default settings, the host header is set anyway to the one of the backend. This can be changed globally, and also in a route specific way with the preseveHost("true") or preserveHost("false") filter. I guess too much details for a 1-minute intro, anyway.